### PR TITLE
Try to fix R build

### PR DESCRIPTION
**Pull request by rcurtin**
I figured I'd give #2966 a shot... I have no idea if I'll have any success. :)

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -38,7 +38,7 @@ jobs:
         - name: Query dependencies
           run: |
             cp src/mlpack/bindings/R/mlpack/DESCRIPTION.in DESCRIPTION
-            Rscript -e "install.packages('remotes')" -e "saveRDS(remotes::dev_package_deps(dependencies = TRUE), 'depends.Rds')"
+            Rscript --verbose -e "install.packages('remotes')" -e "saveRDS(remotes::dev_package_deps(dependencies = TRUE), 'depends.Rds')"
```
Comment (zoq) [inline .github/workflows/main.yml:41]: Do you like to keep the verbose output enabled?

Comment (rcurtin) [reply .github/workflows/main.yml:41]: Only for debugging---I'll remove it once I get things working. :+1:

**Comment by rcurtin** [comment]
Hey, it works!  Looks like all I needed to do was disable the cache on OS X.

**Review by zoq — APPROVED** [review]
I guess we should not start counting the hours we put into random CI build issues :+1:

**Comment by rcurtin** [comment]
Ok, hopefully now we will start seeing green builds! :)

**Pull request merged**